src/main/java/stream/flarebot/flarebot/commands/moderation/NINOCommand.java

```diff
@@ -113,11 +116,10 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                     guild.getNINO().clearRemoveMessages();
                     MessageUtils.sendSuccessMessage("Successfully removed **all** messages from this guild!", channel);
                 } else if (args[1].equalsIgnoreCase("list")) {
-                    //TODO: Move this to pages for 4.2
-                    int page = 1;
+                    int page = 0;
                     if (args.length > 2)
-                        page = GeneralUtils.getInt(args[2], 1);
-                    listMessages(guild, page, channel, sender);
+                        page = GeneralUtils.getInt(args[2], 0);
```

Comment:
You expect users to use `_nino message list 0`? and 1 for page 2?

Reply:
thats allready handle walshy.......... within the sendembedmessage

Reply:
wait nvm I'm a derp